package.json

```diff
@@ -37,9 +37,9 @@
     "clean:l10n": "git push origin --delete l10n_master",
     "pack:dir": "npm run clean:dist && electron-builder --dir -p never",
     "pack:lin": "npm run clean:dist && electron-builder --linux --x64 -p never",
-    "pack:mac": "npm run clean:dist && electron-builder --mac -p never",
-    "pack:mac:mas": "npm run clean:dist && electron-builder --mac mas -p never",
-    "pack:mac:masdev": "npm run clean:dist && electron-builder --mac mas-dev -p never",
+    "pack:mac": "npm run clean:dist && electron-builder --mac --universal -p never",
```

Comment:
This unfortunately doubles the application size, we can't do much about the Mac App Store unfortunately but for `dmg` we should strive towards creating two separate binaries which the users may download.

Reply:
Thank you for the comment. I added a `pack:mac_arm64` target and reverted the `pack:mac` target.

I left the `--universal` target for the Mac App Store.